RELEASE_NOTES.md

```diff
@@ -8,6 +24,64 @@ autoconf   | >=2.61
 zlib1g-dev | >=0.13.x
 libtool    | >=2.x
 
+Migration Instructions
+----------------------
+
+* The generated documentation is now written to a top-level folder `sphinx-build`.
+  Since `clean*` targets have been changed accordingly, documentation generated
+  prior to this release have to be removed manually.
```

Comment:
> his is just a build artefact, it doesn't really affect users, right?
Yes 
Agree, putting it in **Migration Instructions** is a bit misleading. Will put it under **Issues Fixed**